frontend/src/components/PreviewReviewModal.tsx

```diff
@@ -0,0 +1,153 @@
+import {
+  Box,
+  HStack,
+  Image,
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalHeader,
+  ModalOverlay,
+  Stack,
+  Tag,
+  Text,
+} from "@chakra-ui/react";
+import React from "react";
+
+/**
+ * The model defining the props for the Preview Review Modal component
+ */
+interface PreviewReviewModalProps {
+  /** The title of the book that is being reviewed */
+  title: string;
+  /** The subtitle of the book that is being reviewed */
+  subtitle: string;
+  /** The name of the author of the book */
+  writtenBy: string;
```

Comment:
Mb this was not in the ticket, but the intended behaviour (for now) for multiple books is to just take the 0th one and display its info. Is this doable here or should this be a separate PR?

Reply:
probably better in a separate PR because we don't have the designs rn